packages/contentstack-auth/test/unit/commands/tokens-add.test.ts

```diff
@@ -87,7 +76,7 @@ describe('Tokens Add Command', () => {
     inquireStub.restore();
   });
 
-  it('Add a token with invalid api key, should fail to add', async function () {
+  it.skip('Add a token with invalid api key, should fail to add', async function () {
```

Comment:
@almeida-shrutika Any specific reason to skip this test case.?

Reply:
@antonyagustine  Actually, the api-key validation implementation is removed from the code so I have skipped the test case

Reply:
If the current CLI version does not consider this specific case as valid, would it be possible to remove the test instead of skipping it?

Reply:
Sure will remove it